### ISSUE TITLE
Theme Showcase: Showcase Banner Close Button Needs Label

### ISSUE THREAD

**Issue by davidakennedy** [Automattic/wp-calypso#32287]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/themes
2. See the theme showcase banner.
3. Inspect source.
4. Its close button does not have a label.

#### What I expected

I'd expect the button to have some sort of label, likely an [`aria-label` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) since the button is only an icon.

#### What happened instead

The button lacks a label, so it can't be announced in a screen reader.

#### Browser / OS version

All.

#### Screenshot / Video

![Screen Shot 2019-04-15 at 3 20 42 PM](https://user-images.githubusercontent.com/1473618/56159221-1e510180-5f92-11e9-9231-705d79c25809.png)

#### Context / Source

Found this doing some manual accessibility testing with aXe. #manual-testing
